.github/actions/bitcoin-int-tests/Dockerfile.bitcoin-tests

```diff
@@ -27,3 +27,4 @@ RUN cargo test -- --test-threads 1 --ignored tests::should_succeed_handling_malf
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::size_overflow_unconfirmed_microblocks_integration_test
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::runtime_overflow_unconfirmed_microblocks_integration_test
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::antientropy_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::atlas_stress_integration_test
```

Comment:
Can we move this entry to `Dockerfile.atlas-test` instead?

Reply:
Done